src/App.tsx

```diff
@@ -16,35 +18,77 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button type="button" className="button is-info is-light">
-          Sort alphabetically
-        </button>
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setReverse] = useState(false);
+  const [isReset, setReset] = useState(goodsFromServer);
```

Comment:
The `isReset` state is initialized with `goodsFromServer`, which is misleading. It seems like this state is intended to store the original list of goods, but it's not used correctly in the `handleReset` function. Consider renaming this state to something more descriptive and use it to reset the `visibleGoods`.